samples/javascript_nodejs/57.teams-conversation-bot/bots/teamsConversationBot.js

```diff
@@ -7,61 +7,114 @@ const {
     TeamsInfo,
     TeamsActivityHandler,
     CardFactory,
-    ActionTypes
-} = require('botbuilder');
-const TextEncoder = require('util').TextEncoder;
+    ActionTypes,
+} = require("botbuilder");
+const TextEncoder = require("util").TextEncoder;
```

Comment:
```suggestion
} = require('botbuilder');
const TextEncoder = require('util').TextEncoder;
```

Reply:
Marking this as Resolved as wrong file went in this PR